frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp

```diff
@@ -131,116 +139,112 @@
             }
         });
 
-        $(window).load( function() {
-        $('.TogglerModeratorsList').on('click', toggleModeratorsList );
-    });
-
-    function useSearch(searchFieldId) {
-        var searchText = $('#'+searchFieldId).val();
-        moderationsDataTable.search(searchText).draw();
-        closedModerationsDataTable.search(searchText).draw();
-    }
+        function useSearch(searchFieldId) {
+            var searchText = $('#'+searchFieldId).val();
+            moderationsDataTable.search(searchText).draw();
+            closedModerationsDataTable.search(searchText).draw();
+        }
```

Comment:
all non-formattiing changes were made above this line.

Reply:
file format changes reverted for moderation/view.jsp